.github/workflows/Package.yml

```diff
@@ -33,7 +33,7 @@ on:
       requirements:
         description: 'Python dependencies to be installed through pip.'
```

Comment:
Maybe we add here:
`.... If empty, use build requirements listed in pyproject.toml`

Reply:
Done.